spec/_shared/common-specs/base_centos.rb

```diff
@@ -0,0 +1,70 @@
+require 'serverspec'
+
+# Tests shared by all images that inherit some version of the base.
+shared_context 'base_centos' do
+  let(:base_packages) { [
+    'gcc', 
+    'gcc-c++',
+    'make',
+    'openssl-devel',
+    'kernel-devel',
+    'make',
+    'autoconf',
+    'automake',
+    'curl',
+    'nano',
+    'telnet',
+    'net-tools',
+    'rsync',
+    'wget',
+    'git',
+  ] }
+  let(:centos_version) { 'CentOS Linux release 7.3.1611 (Core)' }
```

Comment:
Please remove everything after 7 as the minor and patch version. 
Eg let(:centos_version) { 'CentOS Linux release 7' }